psi/wsgi.py

```diff
@@ -1,20 +1,24 @@
 # coding=utf-8
+import sys
+import os
 
-import sys, os
 
-sys.path.insert(0, os.getcwd() + "/psi")
-
-# This import should behind above two lines
-# To avoid Chinese character display issue
-from psi.app import create_app, init_all
+try:
+    # This import should behind above two lines
+    # To avoid Chinese character display issue
+    from psi.app import create_app, init_all
+except ImportError:
+    # If os.getcwd() is _inside_ the psi package, add parent directory
+    # to front of PATH so the interpreter can import the `psi` package
+    _root_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..')
+    sys.path.insert(0, _root_dir)
+    # Try the import again!
+    from psi.app import create_app, init_all
 
 
 application = create_app()
-socket_io = init_all(application)
+socket_io = init_all(application, migrate=False)
+
 
 if __name__ == '__main__':
     application.run(threaded=True, debug=True)
```

Comment:
I think lines 23 & 24 should be deleted. I would argue that `wsgi.py` shouldn't invoke the debug app--instead we should explicitly invoke that from the flask utility script. Thoughts?

Reply:
Yes,  sounds reasonable :)